src/protocol/templates/pingmessage_.h.in

```diff
@@ -9,47 +9,48 @@
 
 #include "pingmessage.h"
 
-{% set message_type = _actual_message_type %}
-{% for message in messages[message_type]['all_msgs']|sort %}
-{% set total_payload = generator.calc_payload(messages[message_type]['all_msgs'][message].payload) %}
+{% for message_type in messages %}
+{% for message in messages[message_type] %}
+{% set m = messages[message_type][message] %}
```

Comment:
Avoid single letter variables

Reply:
(this file has been removed, I checked the others)